tests/test_core.py

```diff
@@ -189,7 +190,8 @@ class MyObject(object):
 
     assert 'MyObject' == str(value)
     assert '<MyObject>' == repr(value)
-    assert u'обѥѩкт' == unicode(value)
+    if six.PY2:
```

Comment:
Not sur if the u' is needed here , in py3 you test it against the str cast of value, perhaps there is a six function that casts to Unicode in Python and to str in py3?